images/image-user/cloudbuild-manifests.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
```

Comment:
I think we do not need that file in this repository. What is the intention of it?

Reply:
It creates multi-arch images for non-x86 architecture. Refer to #287 for more details.